hs_communities/static/js/my-communities.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function () {
+  const MyCommunitiesApp = new Vue({
+    el: "#my-communities-app",
+    delimiters: ['${', '}'],
+    data: {
+      test: 'test!!',
+      isRemoving: {},
+      targetRequest: null,
+      pendingRequests: PENDING_REQUESTS
+    },
+    mounted: function () {
+    },
+    methods: {
+      remove: async function(id) {
+        this.$set(this.isRemoving, id, true)
+        const url = '/access/_internal/crequest/remove/' + id + '/';
+        try {
+          const response = await $.post(url)
+          if (response.message === "Request removed") {
+            this.pendingRequests = response.pending
+          }
+          this.$set(this.isRemoving, id, false)
+          $("#remove-community-request-modal").modal('hide')
+          customAlert("Cancel Request", response.message, "success", 6000, true);
```

Comment:
more explanation than "Cancel Request"?

Reply:
I think the 'Cancel Request' title is consistent with our messaging where we use the name of the action being performed as the title, and the outcome of the action as the body. The outcome in this case is returned in the response object as `response.message`. I found the messages in this response user friendly enough.

https://github.com/hydroshare/hydroshare/blob/4372-communities-and-groups-2.0/hs_access_control/views.py#L823

Reply:
Good point, thanks